README.md

```diff
@@ -1,7 +1,7 @@
 # Viam Triton ML Model Service
 
 A Viam provided ML Model service backed by NVIDIA's [Triton Inference Server](https://developer.nvidia.com/triton-inference-server).
-Configure this ML Model service as a modular resource on your machine with a [Jetson board](https://docs.viam.com/build/configure/components/board/jetson/) to deploy TensorFlow or PyTorch ML models to your machine faster while consuming less computer power.
+Configure this ML Model service as a modular resource on your machine with a [Jetson board](https://docs.viam.com/build/configure/components/board/jetson/) to deploy TensorFlow (not TensorFlow Lite) or PyTorch ML models to your machine faster while consuming less computer power.
```

Comment:
Triton actually supports several other model types: ONNX, OpenVino, TensorRT. I'm reluctant to try to document them all here, since I don't want us to be in the position of trying to document NVidia's stuff, and the exact set of supported model types potentially varies with version of the module (since that in turn uses a varying version of the Triton server). 

There is this page: https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/backend/docs/backend_platform_support_matrix.html.

But I can't find a versioned link. Also, it is important to note that not all model types are supported on all platforms. For instance, OpenVINO is not supported on Jetpack.

Reply:
```suggestion
Configure this ML Model service as a modular resource on your machine with a [Jetson board](https://docs.viam.com/build/configure/components/board/jetson/) to deploy a [supported ML model](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/backend/docs/backend_platform_support_matrix.html) to your machine faster while consuming less computer power.
```

Good point. Instead of listing them here, I'll just link out -- at least then, users won't see outdated or incorrect information on our side. As long as something under the hood dictates support, it makes sense to link out IMO.

Reply:
Used a separate suggestion to implement this feedback along with another comment's suggestion, we can ignore this one.